### PR TITLE
Add SketchUp

### PR DESCRIPTION
**Pull request by sjstark**
sketchup

**Issue:** #1984
**Alexa rank:** 1,896

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with **SVGO** or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
Selected hex value of "block" part of logo as the prominent color feature of the logo.
Logo was taken from [Wikimedia Commons](https://commons.wikimedia.org/wiki/File:SketchUp_logo.svg), but matches [website](https://www.sketchup.com/).

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -5525,6 +5525,11 @@
             "hex": "F7B500",
             "source": "https://www.sketch.com/press/"
         },
+        {
+            "title": "SketchUp",
+            "hex": "E9292F",
```
Comment (runxel) [inline _data/simple-icons.json:5530]: I'm not sure about this color?
Where did you get it from, exactly?

The color of the Wikimedia icon is different, and so is the icon on the website... (`#F62737`) 🤔

**Review by runxel — COMMENTED** [review]
Icon looks great, not sure with the color tho.

**Comment by runxel** [comment]
Thanks for the work! Looks great! 💯

**Comment by sjstark** [comment]
@runxel you're absolutely correct. The color picker software I was using originally apparently awful (or maybe taking into account my f.lux adjustments at the time?)

Either way, I've updated the color in the file to the #F62737 listed.

(I've also updated my color picker to just a chrome extension)

**Comment by creepertron95** [comment]
@sjstark I was kind of curious of what color picker you used. On my computer, I used a lightweight color picker program called [ColorCop](http://colorcop.net/). It's old but it works for me.
colorcop

**Comment by sjstark** [comment]
The name should have been the first red flag... [Free Color Picker](https://www.microsoft.com/en-us/p/free-color-picker-color-picker-from-screen-html-color-picker-hex-color-picker/9p9qkfj0fh21?activetab=pivot:overviewtab)

I got lazy and picked the first free one I found on the microsoft store. Downloaded the one you linked and like it much more!

**Comment by runxel** [comment]
I personally used the ColorCop too for a long time. But it doesn't work with multi displays. So nowadays I just use [PowerToys](https://github.com/microsoft/PowerToys/), which has also a colorpicker, bound to default `win + shift + c` and it's just working.

**Pull request merged**